--- cov-analysis-linux64-2019.03/config/templates/gnu/compiler-compat-arm-intrin.h	2019-12-12 16:13:12.807998525 -0600
+++ cov-analysis-linux64-2019.03/config/templates/gnu/compiler-compat-arm-intrin_new.h	2019-12-12 16:14:23.559761391 -0600
@@ -1373,11 +1373,11 @@
 typedef signed char int8_t;
 typedef short int int16_t;
 typedef int int32_t;
-typedef long int int64_t;
+typedef signed long long int64_t;
 typedef unsigned char uint8_t;
 typedef short unsigned int uint16_t;
 typedef unsigned int uint32_t;
-typedef long unsigned int uint64_t;
+typedef unsigned long long uint64_t;

 __Int8x8_t __builtin_aarch64_tbl3v8qi(__builtin_aarch64_simd_oi, __Int8x8_t);
 __Float32x2_t __builtin_aarch64_absv2sf(__Float32x2_t);
