diff --git a/bl32/sp_min/aarch32/entrypoint.S b/bl32/sp_min/aarch32/entrypoint.S
index f10296724..70fa4d357 100644
--- a/bl32/sp_min/aarch32/entrypoint.S
+++ b/bl32/sp_min/aarch32/entrypoint.S
@@ -52,7 +52,7 @@ vector_base sp_min_vector_table
 	b	plat_panic_handler	/* Undef */
 	b	sp_min_handle_smc	/* Syscall */
 	b	report_prefetch_abort	/* Prefetch abort */
-	b	report_data_abort	/* Data abort */
+	b	do_panic		/* Data abort */
 	b	plat_panic_handler	/* Reserved */
 	b	plat_panic_handler	/* IRQ */
 	b	sp_min_handle_fiq	/* FIQ */
